crates/viewer/re_viewer/src/app.rs

```diff
@@ -1119,26 +1119,8 @@ impl App {
     fn show_text_logs_as_notifications(&mut self) {
         re_tracing::profile_function!();
 
-        while let Ok(re_log::LogMsg { level, target, msg }) = self.text_log_rx.try_recv() {
-            let is_rerun_crate = target.starts_with("rerun") || target.starts_with("re_");
-            if !is_rerun_crate {
```

Comment:
I missed - do we still have this filter somewhere? We don't want to show `log::info!` from third-party-crates